src/Aquifer.Common/Services/Caching/CachingLanguageService.cs

```diff
@@ -69,13 +85,19 @@ public async Task<IReadOnlyDictionary<string, int>> GetLanguageIdByCodeMapAsync(
             {
                 cacheEntry.SlidingExpiration = s_cacheLifetime;
 
-                var LanguageCodeById = await _dbContext.Languages
-                    .Select(l => new { l.Id, l.ISO6393Code })
-                    .ToDictionaryAsync(l => l.Id, l => l.ISO6393Code, ct);
+                var languageEntityByIdMap = (await _dbContext.Languages
+                        .ToDictionaryAsync(l => l.Id, l => l, ct))
+                    .AsReadOnly();
+
+                var languageCodeByIdMap = languageEntityByIdMap
```

Comment:
I don't think there should be any issues with caching entities in memory because we're not tracking them but I wanted to call it out here in case something comes up.

Reply:
I updated to no longer cache the language entity and use a new record type, intsead.  After letting the code settle for a bit, I think it's best to never cache EF models as a general rule.